modules/technology-preview.adoc

```diff
@@ -8,7 +8,7 @@ are not supported with Red Hat production service level agreements (SLAs) and
 might not be functionally complete. Red Hat does not recommend using them
 in production. These features provide early access to upcoming product
 features, enabling customers to test functionality and provide feedback during
-the development process.
+the development process. The GA is available through Red Hat Virtualization. You must not install both the Kubernetes NMState Operators at the same time.
```

Comment:
```suggestion
the development process. The GA is available through {rh-virtualization-first}. Do not install both the Kubernetes NMState Operators at the same time.
```

Red Ht Virtualization has an attribute in the common-attributes.adoc file for us to use. Just clarified the second sentence for easier reading. :)

Reply:
Done